apps/sports-helsinki/src/domain/search/eventSearch/utils.tsx

```diff
@@ -190,13 +198,21 @@ export const getEventSearchVariables = ({
     }
   };
 
+  const sportsKeywords = [
+    ...new Set( // unique keywords
+      sportsCategories.flatMap(
+        (category) => sportsCategoryData?.[category].keywords ?? []
+      )
+    ),
+  ];
+
   return {
     ...getSearchParam(),
     isFree: isFree || undefined,
     end,
     include,
     keywordOrSet1: SPORT_COURSES_KEYWORDS, // Limit search to sport keywords only
-    keywordOrSet2: [...(keyword ?? [])],
+    keywordOrSet2: [...(keyword ?? []), ...sportsKeywords],
```

Comment:
I wonder if it matters if any/some keywords collide with `SPORT_COURSES_KEYWORDS` keywords on `keywordOrSet1` - and/or should this be a diff of those, or one of these sets just be an union.. But I guess this comes down to how we define the `keywordOrSet1` keywords anyway, but right now they might contain some of these as well.